src/main/java/io/synadia/flink/v0/source/reader/NatsSourceFetcherManager.java

```diff
@@ -92,26 +85,19 @@ void closeFetcher(String splitId) {
         }
     }
 
-    public void acknowledgeMessages(Map<String, List<Message>> cursorsToCommit)
-            throws Exception { //TODO Change to nats exception
+    public void acknowledgeMessages(Map<String, List<Message>> cursorsToCommit) throws Exception { //TODO Change to nats exception
         LOG.debug("Acknowledge messages {}", cursorsToCommit);
 
         for (Map.Entry<String, List<Message>> entry : cursorsToCommit.entrySet()) {
-            String partition = entry.getKey();
-            SplitFetcher<Message, NatsSubjectSplit> fetcher =
-                    getOrCreateFetcher(partition);
-            triggerAcknowledge(fetcher, partition, entry.getValue());
+            String splitId = entry.getKey();
+            SplitFetcher<Message, NatsSubjectSplit> fetcher = getOrCreateFetcher(splitId);
+            triggerAcknowledge(fetcher, splitId, entry.getValue());
```

Comment:
so for open source, we want to keep changes very surgical. I understand the desire to refactor, but unless you are changing the functionality or reusing the functionality, don't refactor it.

Reply:
One thing I do if I do make a change like this, is I do a review myself and comment so other reviewers can see why the change was made.